lineQueueClient/src/components/Header/Header.js

```diff
@@ -1,49 +1,46 @@
 import React from 'react'
-import {Text, View, StyleSheet, Image} from 'react-native';
+import {Text, View, StyleSheet, Image,} from 'react-native';
 
 /**
  * Renders the app header
  */
 const Header = () => {
   let headerStyle = StyleSheet.create({
+    headerContainer: {
+      flex:1,
+      backgroundColor: '#151515',
+      maxHeight: 220,
+      justifyContent: 'flex-start',
+      alignItems:'flex-end',
+      paddingRight: 35,
```

Comment:
Remove static positioning

Reply:
right padding is meant to be always static